packages/frontend/src/components/common/ResourceView/ResourceActionMenu.tsx

```diff
@@ -174,304 +193,354 @@ const ResourceViewActionMenu: FC<ResourceViewActionMenuProps> = ({
+                                    ))}
+
+                                    <Can
+                                        I="create"
+                                        this={subject('Space', {
+                                            organizationUuid:
+                                                user.data?.organizationUuid,
+                                            projectUuid,
+                                        })}
+                                    >
+                                        {spaces.length > 0 ? (
+                                            <Menu.Divider />
+                                        ) : null}
+                                        <Menu.Item
+                                            component="button"
+                                            role="menuitem"
+                                            icon={
+                                                <MantineIcon
+                                                    icon={IconPlus}
+                                                    size={18}
+                                                />
+                                            }
+                                            onClick={() => {
+                                                onAction({
+                                                    type: ResourceViewItemAction.CREATE_SPACE,
+                                                    item,
+                                                });
+                                            }}
+                                        >
+                                            Create new space
+                                        </Menu.Item>
+                                    </Can>
+                                </Menu.Dropdown>
+                            </Menu>
+                        </>
+                    ) : null}
+
+                    {allowDelete && (
+                        <>
+                            <Menu.Divider />
+
+                            <Menu.Item
+                                component="button"
+                                role="menuitem"
+                                color="red"
+                                icon={
+                                    <MantineIcon icon={IconTrash} size={18} />
+                                }
+                                onClick={() => {
+                                    onAction({
+                                        type: ResourceViewItemAction.DELETE,
+                                        item,
+                                    });
+                                }}
+                            >
+                                Delete {item.type}
+                            </Menu.Item>
+                        </>
+                    )}
+                </Menu.Dropdown>
+            </Menu>
+
+            {(promoteChartDiff || promoteChartDiffLoading) && (
```

Comment:
added 2 modals, one for chart diff and another for dashboard diff. Outside of the menu